kimchi/src/circuits/gate.rs

```diff
@@ -43,6 +43,7 @@ impl CurrOrNext {
     /// Compute the offset corresponding to the `CurrOrNext` value.
     /// - `Curr.shift() == 0`
     /// - `Next.shift() == 1`
+
```

Comment:
I don't think it's idiomatic to put a space between a doc comment and a function :o

Reply:
Done https://github.com/o1-labs/proof-systems/pull/752/commits/e74530b7cf67a9cf66787be09ff530dff090630e